community/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/SpatialFunctionsAcceptanceTest.scala

```diff
@@ -185,6 +185,50 @@ class SpatialFunctionsAcceptanceTest extends ExecutionEngineFunSuite with NewPla
     result.toList should equal(List(Map("dist" -> null)))
   }
 
+  test("distance function should return null with null in any kind of point on rhs") {
+    var result = executeWithAllPlanners(
+      """return distance(point({latitude:3,longitude:7}),point({latitude:null,
+        |longitude:3})) as dist;""".stripMargin)
+    result.toList should equal(List(Map("dist" -> null)))
+
+    result = executeWithAllPlanners(
+      """return distance(point({latitude:3,longitude:7}),point({latitude:3,
+        |longitude:null})) as dist;""".stripMargin)
```

Comment:
Choice of the column to wrap on looks a little odd, perhaps one line is better, with normal quotes and no stripMargin?